static/docs/user-guide/howto/external-data-and-cache.md

```diff
@@ -0,0 +1,151 @@
+# Keep Data and Cache Outside the Project
+
+Sometimes we would like to keep the data and cache outside the project
+directory, so that we can search easily the code of the project.
+
+Keeping the data outside of the project, on a fixed absolute path, is also
```

Comment:
it's a a very tricky case - if multiple projects use the same data how do they switch different versions?